Shared/Samples/Apply function to raster from service/README.metadata.json

```
@@ -0,0 +1,32 @@
+{
+    "category": "Layers",
```

```suggestion
    "category": "Visualization",
```
Should this be in the "Visualization" category? Related to this [comment](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/589#discussion_r2076239914).